### ISSUE TITLE
windows: UI error reading base template

### ISSUE THREAD

**Issue by alexandrul** [prometheus/prometheus#4596]
## Bug Report
**What did you do?**
Upgraded to 2.4.0 and opened http://localhost:9090 in Chrome

**What did you expect to see?**
The web UI

**What did you see instead? Under which circumstances?**
error reading base template: open /\templates\_base.html: file does not exist

**Environment**

* System information:
Windows Server 2008 R2

* Prometheus version:
prometheus, version 2.4.0 (branch: HEAD, revision: 068eaa5dbfce6c08f3d05d3d3c0bfd96267cfed2)
  build user:       root@d84c15ea5e93
  build date:       20180911-10:56:06
  go version:       go1.10.3

**Comment by alexandrul** [comment]
stderr:

```
level=info ts=2018-09-12T06:21:31.7431125Z caller=main.go:238 msg="Starting Prometheus" version="(version=2.4.0, branch=HEAD, revision=068eaa5dbfce6c08f3d05d3d3c0bfd96267cfed2)"
level=info ts=2018-09-12T06:21:31.7431125Z caller=main.go:239 build_context="(go=go1.10.3, user=root@d84c15ea5e93, date=20180911-10:56:06)"
level=info ts=2018-09-12T06:21:31.7431125Z caller=main.go:240 host_details=(windows)
level=info ts=2018-09-12T06:21:31.7431125Z caller=main.go:241 fd_limits=N/A
level=info ts=2018-09-12T06:21:31.7431125Z caller=main.go:242 vm_limits=N/A
level=info ts=2018-09-12T06:21:31.7441126Z caller=web.go:397 component=web msg="Start listening for connections" address=0.0.0.0:9090
level=info ts=2018-09-12T06:21:31.7441126Z caller=main.go:554 msg="Starting TSDB ..."
level=info ts=2018-09-12T06:21:31.7521134Z caller=main.go:564 msg="TSDB started"
level=info ts=2018-09-12T06:21:31.7521134Z caller=main.go:624 msg="Loading configuration file" filename=C:\...\prometheus.yml
level=info ts=2018-09-12T06:21:31.7591141Z caller=main.go:650 msg="Completed loading of configuration file" filename=C:\...\prometheus.yml
level=info ts=2018-09-12T06:21:31.7591141Z caller=main.go:523 msg="Server is ready to receive web requests."
```

**Comment by simonpasquier** [comment]
This is probably related to #4430. I'll have a look at it.

**Comment by nadilas** [comment]
+1

**Comment by simonpasquier** [comment]
@alexandrul and @nadilas it would be great if you could test https://github.com/prometheus/prometheus/pull/4616

**Comment by alexandrul** [comment]
@simonpasquier seems fine on my PC (using go 1.11 and latest cygwin64).

**Comment by notOccupanther** [comment]
Still having this issue.  Just downloaded 2.4 this morn. (Windows 10)

Accessing localhost:9090/metrics shows detail, 

but localhost:9090 shows:
"error reading base template: open /\templates\_base.html: file does not exist"

**Comment by simonpasquier** [comment]
@notOccupanther you'll have to wait for v2.4.1 (see https://github.com/prometheus/prometheus/pull/4632).

**Comment by codesome** [comment]
@simonpasquier I think this can be closed now.

**Comment by gouthamve** [comment]
Closed in #4616

**Comment by simonpasquier** [comment]
however the change isn't on master yet which is probably why GitHub didn't auto-close this issue.
